plugins/modules/keycloak_client.py

```diff
@@ -340,6 +340,40 @@
         description:
             - Override realm authentication flow bindings.
         type: dict
+        suboptions:
```

Comment:
Other than these four options specified, are there any other parameters in this `dict()` that will be accepted by keycloak's API?

Reply:
If there are any, or have been in the past, this is a breaking change and cannot be merged without some hoops to jump though.

Reply:
Hi, thanks for your time.  I dont know if this is enough, but the Interface for Client flows was commited 7 years ago and never modified
https://github.com/keycloak/keycloak/blob/main/server-spi/src/main/java/org/keycloak/models/AuthenticationFlowBindings.java

Reply:
Would you prefer that we go back to a generic dictionary and apply the conversion just if the attribute ends with "name"?

Reply:
No, I think it's fine this way.